src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -0,0 +1,209 @@
+        addClass(this.playerContainer, 'jw-tizen-app');
+
+        const element = this.context.createElement('div');
+        element.className = 'jw-tizen-controls jw-tizen-reset';
+        this.div = element;
+    
+        const backdrop = this.context.createElement('div');
+        backdrop.className = 'jw-controls-backdrop jw-reset';
+        this.backdrop = backdrop;
+
+        // Pause Display
+        if (!this.pauseDisplay) {
+            const pauseDisplay = this.context.createElement('div');
+            pauseDisplay.className = 'jw-pause-display';
+
+            const pauseDisplayContainer = this.context.createElement('div');
+            pauseDisplayContainer.className = 'jw-pause-display-container jw-reset-text';
+
+            const pauseTitle = this.context.createElement('div');
+            pauseTitle.className = 'jw-pause-title jw-reset-text';
+
+            const pauseDescription = this.context.createElement('div');
+            pauseDescription.className = 'jw-pause-description jw-reset-text';
+
+            pauseDisplayContainer.appendChild(pauseTitle);
+            pauseDisplayContainer.appendChild(pauseDescription);
+
+            const title = new Title(model);
+            title.setup(pauseDisplayContainer);
+            pauseDisplay.appendChild(pauseDisplayContainer);
+            this.div.appendChild(pauseDisplay);
+            this.pauseDisplay = pauseDisplay;
+        }
+
+        // Display Buttons - Buffering
+        if (!this.displayContainer) {
+            const displayContainer = new DisplayContainer(model, api);
+
+            this.div.appendChild(displayContainer.element());
+            this.displayContainer = displayContainer;
+        }
+
+        // Controlbar
+        const controlbar = this.controlbar = new TizenControlbar(api, model,
+            this.playerContainer.querySelector('.jw-hidden-accessibility'));
+        controlbar.on('backClick', () => {
+            this.onBackClick(api, model);
+        });
+
+        // Next Up Tooltip
+        if (model.get('nextUpDisplay') && !controlbar.nextUpToolTip) {
+            const nextUpToolTip = new NextUpToolTip(model, api, this.playerContainer);
+            nextUpToolTip.setup(this.context);
+            controlbar.nextUpToolTip = nextUpToolTip;
+
+            // NextUp needs to be behind the controlbar to not block other tooltips
+            this.div.appendChild(nextUpToolTip.element());
+        }
+
+        this.div.appendChild(controlbar.element());
+
+        // Settings/Tracks Menu
+        const localization = model.get('localization');
+        this.settingsMenu = new SettingsMenu(api, model.player, this.controlbar, localization);
+
+        const handleKeydown = (evt: KeyboardEvent) => {
+            if (this.controlbar) {
+                this.controlbar.handleKeydown(evt, this.showing);
+            }
+            switch (evt.keyCode) {
+                case 37: // left-arrow
+                    this.userActive();
```

Comment:
you can remove all these cases and just make it a fall-through (unless out lint rules don't allow that)
```
case 37:
case 39:
...etc:
  this.userActive();
  break;
```